packages/flex-plugins-api-utils/src/http.ts

```diff
@@ -15,7 +16,14 @@ export interface AuthConfig {
   password: string;
 }
 
-export interface HttpConfig {
+export interface OptionalHttpConfig {
+  caller?: string;
```

Comment:
should we whitelist the caller to be dashboard or cli?

Reply:
It is whitelisted on the plugins-api role. User-agent can be modified by the customer anyway

Reply:
if it's whitelisted on plugins-api then that's fine, but the caller (not user agent) would be hardcoded in admin plugin and cli, wouldn't it?

Reply:
Caller is added to user-agent, same as the `packages`. As we add more supported `callers` we'll just need to update plugins-api to allow it. This will remain a generic `string`. If you pass `caller: foo` then plugins-api will log it as `unknown`.